Dockerfile

```diff
@@ -11,14 +11,14 @@ ARG CEPH_REPO_URL=https://download.ceph.com/debian-nautilus/
 RUN wget -q -O- 'https://download.ceph.com/keys/release.asc' | apt-key add -
 RUN apt-add-repository "deb ${CEPH_REPO_URL} xenial main"
 
-RUN add-apt-repository -y ppa:gophers/archive
+RUN add-apt-repository -y ppa:longsleep/golang-backports
```

Comment:
I'm not very up to date on my ubuntu-isms. Can we avoid the need for a ppa? I'm trying to avoid my personal desire to switch to a distro I'm more familiar with .... but it seems like we ought not need a bunch of weird addons even for ubuntu right?

Reply:
And regardless of how this pr goes, we should really not be using go 1.10 any more as two go realeases newer are already out.

Reply:
This is what I got from the [golang wiki](https://github.com/golang/go/wiki/Ubuntu). Ubuntu seems to have an old version of Golang, I do not remember what version it was though...